operator/src/main/java/oracle/kubernetes/operator/helpers/PodStepContext.java

```diff
@@ -603,6 +622,13 @@ void overrideContainerWeblogicEnvVars(List<V1EnvVar> vars) {
     if (isLocalAdminProtocolChannelSecure()) {
       addEnvVar(vars, "ADMIN_PORT_SECURE", "true");
     }
+    if (isAdminServerProtocolChannelSecure()) {
```

Comment:
As follow on to the comment above about the method and being overriden, can you add a comment for line 622 about this as well or better yet make it clear the difference between ADMIN_PORT_SECURE and ADMIN_SERVER_PORT_SECURE in that one is for the local servers connection and the other is for the connection to the admin server of the domain.